compiler/luci/pass/src/ResolveCustomOpAddPass.cpp

```diff
@@ -73,19 +129,39 @@ bool resolve_with_BroadcastTo(luci::CircleCustom *addv2)
   if (broadcastTo_idx == -1)
     return false;
 
-  auto input = loco::must_cast<const luci::CircleCustomOut *>(addv2->inputs(broadcastTo_idx));
-  auto broadcastTo = loco::must_cast<luci::CircleCustom *>(input->input());
-
   auto name = addv2->name();
   assert(name.length() > 0);
 
+  auto input = loco::must_cast<const luci::CircleNode *>(addv2->inputs(broadcastTo_idx));
   auto add = addv2->graph()->nodes()->create<luci::CircleAdd>();
   add->fusedActivationFunction(luci::FusedActFunc::NONE);
   add->x(addv2->inputs(1 - broadcastTo_idx));
-  add->y(broadcastTo->inputs(0));
+
+  switch (input->opcode())
+  {
+    case luci::CircleOpcode::BROADCAST_TO:
+    {
+      auto broadcastTo = loco::must_cast<luci::CircleBroadcastTo *>(addv2->inputs(broadcastTo_idx));
+      add->y(broadcastTo->input());
+      luci::add_origin(
+        add, luci::composite_origin({luci::get_origin(broadcastTo), luci::get_origin(addv2)}));
+      break;
+    }
+    case luci::CircleOpcode::CIRCLECUSTOMOUT:
+    {
+      auto inputOut =
+        loco::must_cast<const luci::CircleCustomOut *>(addv2->inputs(broadcastTo_idx));
+      auto broadcastTo = loco::must_cast<luci::CircleCustom *>(inputOut->input());
+      add->y(broadcastTo->inputs(0));
+      luci::add_origin(
+        add, luci::composite_origin({luci::get_origin(broadcastTo), luci::get_origin(addv2)}));
+      break;
+    }
+    default:
+      return false;
```

Comment:
`auto add = addv2->graph()->nodes()->create<luci::CircleAdd>();` is useless for this case.

Reply:
@seanshpark 

> auto add = addv2->graph()->nodes()->create<luci::CircleAdd>(); is useless for this case.

I'm asking because I don't know 😅,
is there a case where `CircleAdd` comes in here?

Reply:
> is there a case where CircleAdd comes in here?

what do you mean by comes in here?

Reply:
@seanshpark 

> what do you mean by comes in here?
At first, I misunderstood `CircleAdd` as input to switch-case statement. 😅

I revised the related code again so that `auto add = addv2->graph()->nodes()->create<luci::CircleAdd>();` can be declared and used. :)